azure-wildfly/src/main/java/sunstone/azure/impl/AzureWFArchiveDeployer.java

```diff
@@ -88,11 +94,32 @@ static void undeployFromWebApp(Identification resourceIdentification, AzureSunst
     }
 
     static void deployToVmInstance(String deploymentName, Identification resourceIdentification, WildFly wildFly, InputStream is, AzureSunstoneStore store) throws SunstoneException {
+        Deploy.Builder builder = new Deploy.Builder(is, deploymentName, false);
```

Comment:
This seems to be simmilar to what is in the `aws-wildfly` module. Wouldn't it be worth it to have a functionality in a util class in `core-wildfly` module?